conversion/conversion.go

```diff
@@ -346,6 +367,7 @@ func dataFromDatabaseForDataflowMigration(targetProfile profiles.TargetProfile,
 	conv.Audit.StreamingStats.ShardToDataStreamNameMap = make(map[string]string)
 	conv.Audit.StreamingStats.ShardToPubsubIdMap = make(map[string]internal.PubsubCfg)
 	conv.Audit.StreamingStats.ShardToDataflowInfoMap = make(map[string]internal.ShardedDataflowJobResources)
+	conv.Audit.StreamingStats.ShardToGcsResources = make(map[string]internal.GcsResources)
```

Comment:
Nit: `GCS` is an acronym and should be in capitals.

Reply:
I think keeping gcs in capital reduces readability
Also if we look into how other golang libraries have done it, example datastream, we can see they also use Gcs not GCS as given [here](https://pkg.go.dev/cloud.google.com/go/datastream@v1.10.3/apiv1/datastreampb#ConnectionProfile.GetGcsProfile)